lib/ansible/module_utils/distro/_distro.py

```diff
@@ -94,7 +94,8 @@
     'lsb-release',
     'oem-release',
     _OS_RELEASE_BASENAME,
-    'system-release'
+    'system-release',
+    'plesk-release'
```

Comment:
Please add a note here that we have modified the upstream package and maybe link to the upstream PR as well.